_targets.R

```diff
@@ -9,9 +9,11 @@ source("3_visualize.R")
 
 dir.create("2_process/out/", showWarnings = FALSE)
 dir.create("3_visualize/out/", showWarnings = FALSE)
+dir.create("3_visualize/out/daily_timeseries_png/",showWarnings = FALSE)
+dir.create("3_visualize/out/hourly_timeseries_png/",showWarnings = FALSE)
```

Comment:
Responding to your question in the PR, I think it's fine to write all of these. I like having them as a quick reference.

Reply:
Sounds good, thanks! They're meant to be a quick reference but we could obviously change the formatting if anyone's opposed to ggplot defaults : )